### ISSUE TITLE
NullRef in HighlightUrlExtension

### ISSUE THREAD

**Issue by KirillOsenkov** [mono/monodevelop#5665]
7.7 build 805

```
System.NullReferenceException: Object reference not set to an instance of an object
  at MonoDevelop.Ide.Editor.Extension.HighlightUrlExtension+<>c__DisplayClass12_1.<RunUpdateTask>b__1 () [0x00029] in /Users/vsts/agent/2.138.4/work/1/s/monodevelop/main/src/core/MonoDevelop.Ide/MonoDevelop.Ide.Editor.Extension/HighlightUrlExtension.cs:147 
  at MonoDevelop.Core.Runtime+<>c.<RunInMainThread>b__40_0 (System.Object state) [0x00013] in /Users/vsts/agent/2.138.4/work/1/s/monodevelop/main/src/core/MonoDevelop.Core/MonoDevelop.Core/Runtime.cs:344 

```

> VS bug [#662552](https://devdiv.visualstudio.com/DevDiv/_workitems/edit/662552)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.8.0.160 (master)

Pull Request #5674 merged by: Mike Krüger
Author: mono
Commit: b89c8a78570f8370a62a1503241f474c372f9919 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 7.7.0.1070 (release-7.7)

Pull Request #5674 merged by: Mike Krüger
Author: mono
Commit: b89c8a78570f8370a62a1503241f474c372f9919 (mono/monodevelop)

**Comment by xamarin-release-manager** [comment]
Fixed in version 8.6.0.802 (master)

Pull Request #5674 merged by: Mike Kr??ger
Author: mono
Commit: b89c8a78570f8370a62a1503241f474c372f9919 (xamarin/monodevelop)
